onedrive_client/od_auth.py

```diff
@@ -117,7 +117,7 @@ def get_profile(self):
         data = response.json()
         if response.status_code != requests.codes.ok:
             raise ValueError(
-                f"Failed to read user profile:{data['error']['message']}")
+                "Failed to read user profile:{data['error']['message']}")
```

Comment:
This is not a syntax error, this is using the [f-string](https://www.python.org/dev/peps/pep-0498/).


I've just saw now that it requires python 3.6+. Please change to the normal `format` syntax:

```python
'Failed to read user profile: {}'.format(data['error']['message'])
```